src/javascript/app_2/Utils/Date/__tests__/date_time.spec.js

```diff
@@ -1 +1,89 @@
-import * as DateTime from '../date_time';
+import { expect } from 'chai';
+import * as DateTime from '../date_time.js';
+import moment from 'moment';
+
+describe('toMoment', () => {
+    it('return utc epoch value date based on client epoch value passed', () => {
+        const epoch = 1544756041;
+
+        expect(DateTime.toMoment(1544756041)).to.deep.equal(moment.unix(epoch).utc());
+    });
+});
+
+describe('convertToUnix', () => {
+    const setTime = (moment_obj, time) => {
+        const [hour, minute, second] = time.split(':');
+        moment_obj.hour(hour).minute(minute || 0).second(second || 0);
+
+        return moment_obj;
+    };
+
+    it('return correct unix value when date and time passed', () => {
+        const date_epoch = 1544745600;
+        const time = "12:30";
+
+        expect(DateTime.convertToUnix(date_epoch, time)).to.deep.equal(setTime(DateTime.toMoment(date_epoch), time).unix());
+    });
+});
+
+describe('toGMTFormat', () => {
+    it('return correct GMT value when no argument passed', () => {
+        expect(DateTime.toGMTFormat()).to.deep.equal(moment().utc().format('YYYY-MM-DD HH:mm:ss [GMT]'));
+    });
+
+    it('return correct GMT value when argument passed', () => {
+        const time_epoch = 1544757884620;
+        expect(DateTime.toGMTFormat(time_epoch)).to.deep.equal(moment(time_epoch).utc().format('YYYY-MM-DD HH:mm:ss [GMT]'));
+    });
+});
+
+describe('formatDate', () => {
+    const date_format = "YYYY-MM-DD"
+    it('return correct response when no argument passed', () => {
+        expect(DateTime.formatDate()).to.eql('Invalid date');
+    });
+
+    it('return correct date value when argument passed', () => {
+        // get today date
+        const date = moment();
+        expect(DateTime.formatDate(date, date_format)).to.deep.equal(moment(date).format(date_format));
+    });
+})
+
+describe('daysFromTodayTo', () => {
+    it('return empty string when there is no argument passed', () => {
+        expect(DateTime.daysFromTodayTo()).to.eql('');
+    });
+
+    it('return empty string if the user selected previous day', () => {
+        //get previous day
+        const date = moment().subtract(1, 'days').format("YYYY-MM-DD");
+        expect(DateTime.daysFromTodayTo(date)).to.eql('');
+    });
+
+    it('return difference value between selected date and today', () => {
+        //get date three days from now
+        const date = moment().add('3', 'days').format("YYYY-MM-DD");
+        const diff = moment(date).utc().diff(moment().utc(), 'days');
+        expect(DateTime.daysFromTodayTo(date)).to.deep.equal(diff + 1);
```

Comment:
i think it's better to check with 3 instead of `diff + 1`  because we expected the days would be exactly 3 from today. there's no need to add the logic in line 67 (that should be taken care by `daysFromTodayTo()` itself)